packages/material-ui/src/locale/index.ts

```diff
@@ -1,4 +1,36 @@
-export const azAZ = {
+interface Localization {
```

Comment:
Consider exporting this type. It will be useful for typescript users.

Reply:
Done